compaction/task_manager_module.hh

```diff
@@ -112,6 +112,26 @@ protected:
     virtual future<> run() override = 0;
 };
 
+class rewrite_sstables_keyspace_compaction_task_impl : public rewrite_sstables_compaction_task_impl {
+private:
```

Comment:
perhaps a more descriptive name. upgrade_sstables_compaction_task, which inherits from the base class rewrite_sstables...

this follows the convention in compaction manager.